spec/controllers/people_controller_spec.rb

```diff
@@ -497,6 +497,11 @@
       get :contacts, :person_id => eve.person.to_param
       expect(response.body).to include '"photos":{"count":16}' # eve is not sharing with alice
     end
+
+    it "returns a 406 for json format" do
+      get :contacts, :person_id => 'foo', format: :json
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.